docs/VERSION_UPGRADE.md

```diff
@@ -6,6 +6,29 @@ weight: 3
 
 This doc contains important information for upgrading uds-identity-config versions. It is not meant to be an exhaustive list of changes between versions, rather information and steps required to manually upgrade versions without a full redeploy of keycloak.
 
+## v0.5.1 to v0.5.2
+
+<details open>
+<summary>Upgrade Details</summary>
+
+* An custom Keycloak event logger that replaces the default event logger is [included in this release](https://github.com/defenseunicorns/uds-identity-config/blob/v0.5.2/src/realm.json#L1669), if you wish to enable manually as part of an upgrade do the following (in the `Unicorn Delivery Service` realm):
+  -  Click on the `Realm Settings` > `Events` and add `jsonlog-event-listener`. 
+  - Remove the built in `jboss-logging` event listener.
```

Comment:
Was the switch from "custom-registration-listener" to "registration-event-listener" also relevant to document here? I wasn't as familiar with that change.

Reply:
I think this was just a name change,  but yea that should probably be called out cause I think the event listener would have to be added under the new name if it was an upgrade.